cnxarchive/tests/data/data.sql

```diff
@@ -557,6 +572,7 @@ INSERT INTO module_files VALUES (4, 23, 'PhET_Icon.png');
 INSERT INTO module_files VALUES (6, 23, 'PhET_Icon.png');
 INSERT INTO module_files VALUES (9, 23, 'PhET_Icon.png');
 INSERT INTO module_files VALUES (10, 23, 'PhET_Icon.png');
+INSERT INTO module_files VALUES (20, 109, 'index.cnxml.html');
```

Comment:
A composite module returns 404 if it is requested outside of a collection.  See db1f9b8